scripts/refresh-kani-proptest.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+# Copyright Kani Contributors
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+
+# This file checks that the proptest rlib and symtab are up to are up
+# to date compared to the kani binary. If not, `cargo kani` is called
+# on proptest to refresh the symtab and rlib. This script is not meant
+# to called manually, but rather from `scripts/kani` and
+# `scripts/cargo-kani`.
+
```

Comment:
For a script. Should we set `-ex`?

Reply:
Since both `cargo-kani` and `kani` is using `-eu` I am going with that one.

Also, `-x` prints every command that is run, breaking the "expected" cases

Reply:
Implemented with https://github.com/model-checking/kani/pull/1435/commits/3b0f4751eb7f9b3a45c0327c8c0ac6fd971fb0aa